app/src/main/java/ch/epfl/cs311/wanderwave/model/data/Profile.kt

```diff
@@ -11,4 +12,24 @@ data class Profile(
     var profilePictureUri: Uri? = null,
     var spotifyUid: String,
     var firebaseUid: String,
-)
+) {
+  companion object {
+    fun from(documentSnapshot: DocumentSnapshot): Profile? {
```

Comment:
In this function, we could consider returning null if any of the members do not exist, instead of hardcoded default values

Reply:
@KlaKalma

Reply:
Why not